### PR TITLE
add SDK sample for .Net

### PR DESCRIPTION
**Pull request by TakaValley**
add SDK sample for .Net

### CODE REVIEW

File: SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_ExtractLayout.cs [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                DocumentTable table = result.Tables[i];
+
+                Console.WriteLine($"Table {i} has {table.RowCount} rows and {table.ColumnCount} columns.");
+
+                foreach (DocumentTableCell cell in table.Cells)
+                {
+                    Console.WriteLine($"  Cell ({cell.RowIndex}, {cell.ColumnIndex}) is a '{cell.Kind}' with content: {cell.Content}");
+                }
+            }
+
```
Comment (luzhang06) [inline SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_ExtractLayout.cs:143]: Please add figure extraction as well

File: SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+#endif
+
+            Operation<AnalyzeResult> operation = await client.AnalyzeDocumentAsync(WaitUntil.Completed, "prebuilt-invoice", content);
+            AnalyzeResult result = operation.Value;
+
+            // To see the list of all the supported fields returned by service and its corresponding types for the
+            // prebuilt-invoice model, see:
+            // https://aka.ms/azsdk/formrecognizer/invoicefieldschema
+
+            // Extract DocumentField:
```
Comment (luzhang06) [inline SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs:62]: please test taxW2 form for your code as well. I vaguely remember there's object defined in W2 schema. Make sure your code works for that.

File: SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
+﻿// coding: utf - 8
+// --------------------------------------------------------------------------
+// Copyright(c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// --------------------------------------------------------------------------
+
+// Symbol to instruct that the example analyzes local document file.
+// If want to analyze the document file by url, just update below line to: #undef ANALYZE_LOCAL_FILE
```
Comment (luzhang06) [inline SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs:9]: Add some URL to reference the full schema for each prebuilt model

File: SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs (hunk 2 above)
Comment (TakaValley) [reply SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs:62]: Fixed. Thanks

File: SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_ExtractLayout.cs (hunk 1 above)
Comment (TakaValley) [reply SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_ExtractLayout.cs:143]: fixed

File: SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs (hunk 3 above)
Comment (TakaValley) [reply SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs]: Fixed thx

**Review by luzhang06 — APPROVED** [review]

**Pull request merged**